internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -467,6 +467,11 @@ func (ms *metadataStorage) setIndex(ctx context.Context, index string, key Index
 	pgTable = metadata.table
 	pgIndex = indexNameToPgIndexName(ms.collection, index)
 
+	switch unique.(type) {
+	case nil:
```

Comment:
Where does this `nil` come from? It's not bson type, I'm curious about this special nil handling :thinking:

Reply:
ok `nil` is no longer in the code base.